### PR TITLE
Fix genproto path error:./server/wal/walpb

### PR DESCRIPTION
**Pull request by LeoYang90**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

**Comment by serathius** [comment]
This PR does 2 things: fixes path and adds protoc installation script. I think we should keep bug fixes separate from adding feautres. Could you please move potoc installation to a separate PR?

**Comment by LeoYang90** [comment]
> This PR does 2 things: fixes path and adds protoc installation script. I think we should keep bug fixes separate from adding feautres. Could you please move potoc installation to a separate PR?

ok

File: scripts/genproto.sh [hunk 1]
```diff
@@ -33,8 +33,10 @@ echo "  - gogoproto-root:          ${GOGOPROTO_ROOT}"
 echo "  - grpc-gateway-root:       ${GRPC_GATEWAY_ROOT}"
 GOGOPROTO_PATH="${GOGOPROTO_ROOT}:${GOGOPROTO_ROOT}/protobuf"
 
+go get -u golang.org/x/tools/cmd/goimports
```
Comment (serathius) [inline scripts/genproto.sh:36]: Why you are adding this line?

**Comment by LeoYang90** [comment]
@serathius

**Pull request merged**